packages/forwarder-helper/src/forwarder_helper_factory.ts

```diff
@@ -22,4 +31,220 @@ export const forwarderHelperFactory = {
         const helper = new ForwarderHelperImpl(config);
         return helper;
     },
+    /**
+     * Given a desired makerAsset and SRA url, get a ForwarderHelper
+     * @param   makerAssetData      An array of objects conforming to SignedOrder. Each order should specify the same makerAssetData and takerAssetData
+     * @param   sraUrl              A url pointing to an SRA v2 compliant endpoint.
+     * @param   rpcUrl              A url pointing to an ethereum node.
+     * @param   networkId           The ethereum networkId, defaults to 1 (mainnet).
+     * @return  A ForwarderHelper, see type for definition
+     */
+    async getForwarderHelperForMakerAssetDataAsync(
+        makerAssetData: string,
+        takerAddress: string,
+        sraUrl: string,
+        rpcUrl?: string,
+        networkId: number = 1,
+    ): Promise<ForwarderHelper> {
+        assert.isHexString('makerAssetData', makerAssetData);
+        assert.isETHAddressHex('takerAddress', takerAddress);
+        assert.isWebUri('sraUrl', sraUrl);
+        if (!_.isUndefined(rpcUrl)) {
+            assert.isWebUri('rpcUrl', rpcUrl);
+        }
+        assert.isNumber('networkId', networkId);
+        // create provider
+        const providerEngine = new Web3ProviderEngine();
+        if (!_.isUndefined(rpcUrl)) {
+            providerEngine.addProvider(new RPCSubprovider(rpcUrl));
+        }
+        providerEngine.start();
+        // create contract wrappers given provider and networkId
+        const contractWrappers = new ContractWrappers(providerEngine, { networkId });
```

Comment:
we might be able to extract out the contractWrappers creation into a different factory to make this function a bit shorter, we will also want to use the wrappers to validate orders that don't come from an SRA endpoint